cm/base/BaseCM/cm_input.py

```diff
@@ -1,8 +1,80 @@
 import os
 
+import pyproj
+from shapely.geometry import shape
+from shapely.ops import cascaded_union, transform
+
+import rasterio
+from BaseCM.cm_output import validate
+from rasterstats import zonal_stats
+
+GEOJSON_PROJ = "EPSG:4326"
+
 
 def get_raster_path(raster_name):
     """Returns the path to the raster file based on the raster name."""
     raster_dir = os.path.join(os.environ["UPLOAD_DIR"], "raster")
     raster_path = os.path.join(raster_dir, raster_name, "raster.tiff")
     return raster_path
+
+
+class StatNotComputeError(Exception):
+    """Exception thrown for a non-existing calculation module"""
+
+    pass
+
+
+def validate_selection(
+    selection: dict,
+    raster: str,
+):
+    """
+    Find the number of count for given a raster and a area selection in order to define if the
+    selection is valid or not.
+
```

Comment:
What is a count?

Reply:
The count is a non-null pixel of a raster.. I add this information in the docstring